src/main/java/com/marklogic/client/ext/document/DocumentWriteOperationBuilder.java

```diff
@@ -16,10 +16,18 @@
 
 	private ContentIdExtractor contentIdExtractor = new DefaultContentIdExtractor();
 
-	public DocumentWriteOperation build(AbstractWriteHandle content) {
+	public DocumentWriteOperation build(AbstractWriteHandle content, String topic, Boolean addTopicToCollections ) {
```

Comment:
This isn't a Kafka-specific class (note the package it's in - I'm hoping to port this to ml-javaclient-util once it matures a bit more in this project), so we don't want Kafka-specific parameters here. 

I think the best approach here is to add a method to DocumentWriteOperationBuilder - build(AbstractWriteHandle, DocumentMetadataHandle). That gives a client a chance to construct a DocumentMetadataHandle and populate it a bit themselves before more permissions/collections/etc are added to it. 

Then build(AbstractWriteHandle) can just call build(AbstractWriteHandle, new DocumentMetadataHandle()). 

That gives you a way to add Kafka-specific collections without adding Kafka-specific code to this class.